core/src/main/resources/hudson/model/View/sidepanel.jelly

```diff
@@ -43,6 +45,52 @@ THE SOFTWARE.
         <l:task href="${rootURL}/${it.viewUrl}newJob" icon="icon-new-package icon-md" it="${app}" permission="${it.itemCreatePermission}" title="${%NewJob(it.newPronoun)}"/>
       </st:include>
 
+      <l:task href="#modal" icon="icon-new-package icon-md" title="New Item Popup"/>
+
+      <div class="remodal" data-remodal-id="modal">
+        <div class="container" id="add-item-panel" style="display: none;">
+          <div class="row">
+            <div class="col-md-offset-2 col-md-20">
+              <form method="post" action="createItem" name="createItem" id="createItem">
+                <div class="header">
+                  <div class="add-item-name">
+                    <label for="name">${%ItemName.label}</label>
```

Comment:
Yes, that was it. I fixed the problem by adding newJob properties to sidepanel.properties